pytorch_toolkit/ote/ote/modules/exporters/mmdetection.py

```diff
@@ -38,9 +39,36 @@
 class MMDetectionExporter(BaseExporter):
     def __init__(self):
         super(MMDetectionExporter, self).__init__()
+        self.opset = 10
+
+    @staticmethod
+    def _get_update_config_str(args):
+        update_config = MMDetectionArgsConverter.get_classes_extra_args(args)
+        update_config = ' '.join([f'{k}={v}' for k, v in update_config.items()])
+        update_config = f' --update_config {update_config}' if update_config else ''
+        update_config = update_config.replace('"', '\\"')
```

Comment:
Is it possible to use `shlex.quote`?

Something like that (I'm not sure, but seems it could be really better)
```suggestion
        update_config = f' --update_config {shlex.quote(update_config)}' if update_config else ''
```

Reply:
does not work